core/meta/src/TListOfEnums.cxx

```diff
@@ -183,6 +183,22 @@ TEnum *TListOfEnums::Find(DeclId_t id) const
    return (TEnum *)fIds->GetValue((Long64_t)id);
 }
 
+////////////////////////////////////////////////////////////////////////////////
+/// Specialize FindObject to do search for the
+/// a enum just by name or create it if its not already in the list
+
+TObject *TListOfEnums::FindObject(const char *name) const
```

Comment:
Do other overloads of FindObject also add? I don't remember that... Maybe consider renaming to FindOrAdd()?

Reply:
They should :) ... at least for this (type) of container.  Semantically 'nothing' is added, it is just delayed loading.   This is a trade-off between pre-loading all enums (and actively harvesting them at each transaction) and this type of gymnastics.  (i.e. the cost of avoiding this const_cast is added __unused__ memory alllocation/retention and added (unused) run-time cost)